### ISSUE TITLE
New improved README

### ISSUE THREAD

**Issue by Davejkane** [Davejkane/riv#60]
Change the README to be user focused instead of a call to green developers (I'm happy with the little team that we have, any more at this stage would just be too much getting in each others way).

- [ ] Tidy up the manual. 
- [ ] Emphasis on features.
- [ ] Gif/Screenshot of the program in action.
- [ ] Clearer installation instructions.
- [ ] Remove calls for developer participation

**Comment by gurgalex** [comment]
>  Remove calls for developer participation

I liked the `contributing` section. Seemed welcoming to me despite having no prior experience with SDL2 or image viewers.  Maybe removing the sentence
>  I have purposely left some features unimplemented before open sourcing with the idea that someone can pick them up as a good first contribution

**Comment by Davejkane** [comment]
Yeah, ok. I'll leave some of the contributing section there, and maybe move the rest to the CONTRIBUTING.md

**Comment by NickHackman** [comment]
If we're moving towards a more user facing perspective we should have a man page in addition to the `readme`

**Comment by Davejkane** [comment]
@NickHackman, agreed, but we'll wait with that until we're ready to look at platform specific build scripts. And that's not just yet. 

Unless this works...
https://github.com/clap-rs/clap_generate

**Comment by gurgalex** [comment]
So, we could have `riv` view the rust logos for the screenshot.
It is the `Rust Image Viewer`
![riv_rust_logo](https://user-images.githubusercontent.com/28968644/58304287-4580c700-7db9-11e9-8e90-dcd175abe5fb.png)

**Comment by Davejkane** [comment]
Yeah, was thinking something similar

**Comment by gurgalex** [comment]
Though, we really need to make it clear that `riv` is a minimal GUI image viewer and does not render to the terminal.

**Comment by gurgalex** [comment]
So, the animated gif should probably give a quick demonstration of the important features correct?

Similar to [this interactive Postgres prompt](https://raw.githubusercontent.com/dbcli/pgcli/master/screenshots/pgcli.gif)
